kt/godot-coroutine-library/src/main/kotlin/godot/coroutines/GodotCoroutine.kt

```diff
@@ -2,15 +2,18 @@ package godot.coroutines
 
 import kotlinx.coroutines.CoroutineScope
 import kotlinx.coroutines.CoroutineStart
-import kotlinx.coroutines.Dispatchers
 import kotlinx.coroutines.SupervisorJob
 import kotlinx.coroutines.launch
 import kotlin.coroutines.CoroutineContext
 
 object GodotCoroutine : CoroutineScope {
-    override val coroutineContext  = Dispatchers.Default + SupervisorJob()
+    override val coroutineContext  = GodotDispatchers.ThreadPool + SupervisorJob()
 }
 
-fun godotCoroutine(context: CoroutineContext = Dispatchers.Default, start: CoroutineStart = CoroutineStart.DEFAULT, block: suspend CoroutineScope.() -> Unit) {
+fun godotCoroutine(
+    context: CoroutineContext = GodotDispatchers.ThreadPool,
```

Comment:
The coroutineContext already has the threadDispatcher so you can replace this function argument wit EmptyContext

Reply:
done